build/bin/network-validator.go

```diff
@@ -1,13 +1,17 @@
 package main
 
 // Usage
-// $ network-validator --timeout=1s --config=config/config.yaml
+// $ AWS_REGION=us-east-1  ./network-validator --timeout=3s --config=config/config.yaml --no-tls
```

Comment:
using _--no-tls_ is optional, By default it's false, It expected to be working against tls if no proxy is given